pages/_app/index.jsx

```diff
@@ -80,9 +77,11 @@ class App extends NextApp {
 
   fetchUser = async () => {
     try {
-      const { dataProvider, activity } = new Route(window.location.pathname)
-      await this.store.init(dataProvider, activity)
+      const { router } = this.props
+      await this.store.init(router.query['data-provider-id'])
```

Comment:
data-provider-id is undefined for some reason. Try to access some page directly http://core.dashboard:3000/data-providers/86/overview

Reply:
basically the same as https://github.com/oacore/dashboard/pull/297#issuecomment-624646926

Reply:
Should be resolved now. Could you please check one more time?